_topic_maps/_topic_map.yml

```diff
@@ -2838,7 +2838,7 @@ Topics:
       File: about-ocp-monitoring
     - Name: Monitoring stack architecture
       File: monitoring-stack-architecture
-    - Name: Understanding the monitoring stack - key concepts
+    - Name: Key concepts
```

Comment:
I don't think this is rendering how you intend: 
![image](https://github.com/user-attachments/assets/af42a269-38ad-4ab1-88ee-8096347ac38c)

I'm not sure why. Have you checked local previews?

Reply:
https://87633--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/about-ocp-monitoring/key-concepts

Reply:
I see your comment 0 says you ran the Prow smoke tests, so I'm not sure why this isn't rendering correctly 🤔 I was thinking maybe its one of the IDs, but that would usually throw an error.